### ISSUE TITLE
Move visual elements in the Grants modal for better usability

### ISSUE THREAD

**Issue by chrisxkeith** [usdigitalresponse/usdr-gost#603]
Current:
![image](https://user-images.githubusercontent.com/3921494/201994748-8bf7abf6-800b-4f25-bb23-59974ccb3088.png)
Proposed:
![image](https://user-images.githubusercontent.com/3921494/201995351-539b0b26-3bb5-40af-9845-a0373f391bee.png)

**Comment by mhuang-usdr** [comment]
@rachel699 @cajinajr Any opinions? Thumbs up?

**Comment by rachel699** [comment]
@chrisxkeith @mhuang-usdr Yes, definitely improved layout by better grouping related content together!

**Comment by mhuang-usdr** [comment]
Wait for FTE UXD as part of larger redesign
